pkg/webhook/webhook.go

```diff
@@ -110,28 +115,39 @@ func (w *Webhook) ServeHTTP(rw http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	logrus.Debugf("Webhook payload %+v", payload)
```

Comment:
How often does this log?

Reply:
This is debug log, so won't get logged if you are not on debug mode.